thorlcr/msort/tsorts1.cpp

```diff
@@ -371,7 +371,7 @@ protected: friend class CSortMerge;
                     Owned<ISecureSocket> ssock = secureContextServer->createSecureSocket(socket.getClear());
                     int tlsTraceLevel = SSLogMin;
                     unsigned sortTraceLevel = slave.queryTraceLevel();
-                    if (sortTraceLevel >= MPVerboseMsgThreshold)
+                    if (sortTraceLevel > ExtraneousMsgThreshold)
```

Comment:
as per last comment

Reply:
MPVerboseMsgThreshold was 110, so it was higher than ExtraneousMsgThreshold.

Reply:
ok, I made it >= ExtraneousMsgThreshold